bin/do_deltas.py

```diff
@@ -154,6 +155,18 @@ def cont_fit(data):
             print(" Error while reading flux_calib file {}".format(args.flux_calib))
             sys.exit(1)
 
+    ### Correct multiplicative pipeline inverse variance calibration
+    if (args.ivar_calib is not None):
+        try:
+            vac = fitsio.FITS(args.ivar_calib)
+            ll  = vac[2]['LOGLAM'][:]
+            eta = vac[2]['ETA'][:]
+            forest.correc_ivar = interp1d(ll,eta,fill_value="extrapolate")
+
```

Comment:
@londumas note that the eta interpolation was recently changed to kind="nearest". Can you change that here too?

Reply:
whar is the difference ? Do you want me to change it too for What is the difference ? Do you want me to change it too for flux-calib ?

Reply:
is there a reason why it is not done here ?
```
forest.mean_cont = interp1d(ll_rest[wmc>0.], forest.mean_cont(ll_rest[wmc>0.]) * mc[wmc>0.], fill_value = "extrapolate")
```

Reply:
I am not sure it is better to have "nearest", it seems more natural to have "linear"

Reply:
I agree it's more natural, but the nearest option is more stable when the nieghboring bins are empty or very noisy, and we need stability there. That's not that much of an issue for the mean cont or the mean abs because bins have a very large occupation and the edges are not very important.